### DISCUSSION TITLE
API V3 scoops (shop_w) lost after use store normally

### DISCUSSION THREAD

**Discussion by samikoja** [etsy/open-api#1009] [Q&A] [answered]
The scope (shop_w) lost for many stores so they can't call any function need this scoop. This store working normally so they can create shipping rule, getShipping rule and so on, but for no reason this scoop lost and this api return status code 400 with response: {"error":"insufficient_scope","error_description":"Insufficient scope"} 
Please if you have any solution on this!

Thank you in advance for your support.

**Comment by Gareth064** [comment]
Morning @samikoja, this is something you will need to talk to the Etsy Support team about. I have seen people mention things like this before and Etsy have said they did something by accident in the backend. Maybe this is one of them.

Send an email to developer@etsy.com explaining the problem. And if you get a response, and it fixed, then pop back here and let us know what the issue was. That will help other community members if they come across the same problem.

**Comment by samikoja** [comment, marked as answer]
So, after connecting with Etsy support team, the solution is to disconnect the app and reconnected again.

**Reply by Gareth064** [reply]
I thought that might be there answer. This is fine if you are the only user of your app. But if you have a large customer base using your app then asking them all to reauthenticate could be a pain.

**Reply by samikoja** [reply]
Yes, and this is my case I have around to 13k of customers so, I will find a way to solve issue without making this so pain.
